hyperspy/drawing/_markers/point.py

```diff
@@ -87,7 +87,7 @@ def update(self):
                                  self.get_data_position('y1')])
         self.marker._sizes = [self.get_data_position('size')]
 
-    def plot(self):
+    def plot(self, fig_draw_animated=True):
```

Comment:
I think some of the marker functions can be made private, since the average user will have no reason to call `update` and `plot`, and possible things like `get_data_position`, `add_data`. These could rather be added as properties like `marker.x`, `marker.y`, ...

 But I'm guessing making them private would break the API.